db/migrate/20210809170951_add_machine_data_to_extracts.rb

```diff
@@ -0,0 +1,6 @@
+class AddMachineDataToExtracts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :extracts, :machine_data, :boolean
```

Comment:
will we be searching on this data via the API? e.g. `Extract.where.not(machine_data: nil)` etc? Wondering about if this needs a sparse index or not? 

The extracts table is big so using a default `false` value (serialized from DB if missing since PG 11) and not backfilling would suffice here. That avoids having to rewrite the big table and saves on disk space. 

A weirder alternative would be to signal this machine_data in the `classification_id` column by using something like `0|-1` value in that column. then you can add a method to the Extract model to reflect on this to signal this `machine_data` link
```
# in Extract model
def machine_data
  [0].includes?(classification_id)
end
```
end